lib/ood_core/job/adapters/pbspro.rb

```diff
@@ -85,9 +85,9 @@ def initialize(host: nil, pbs_exec: nil, bin_overrides: {})
           # @param id [#to_s] the id of the job
           # @raise [Error] if `qstat` command exited unsuccessfully
           # @return [Array<Hash>] list of details for jobs
-          def get_jobs(id: "")
-            args = ["-f"]   # display all information
-            args += ["-t"]  # list subjobs
+          def get_jobs(id: "", user: "")
+            args = ["-f", "-t"]   # display all information
+            args += ["-u", user] unless user.empty?
```

Comment:
The new user option doesn't appear to be used anywhere... is there a plan to?

Reply:
It's a remnant of a plan to re-write `info_where_owner` that I abandoned because it would have changed behavior. I'll remove it.